bundles/org.openhab.binding.networkupstools/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:ups:config">
+		<parameter name="device" type="text" required="true">
```

Comment:
Is this device used for something or only as a reference?

Reply:
The device name is send to NUT server to get the variable values for a specific device. I could use the thing id :stuck_out_tongue_winking_eye: